File: 알고리즘/Reverse String.py
class Solution:
    def reverseString(self, s: List[str]) -> None:
        """
        Do not return anything, modify s in-place instead.
        """
        for i in range((len(s)//2)):
            b = len(s)-1-i
            s[i], s[b] = s[b], s[i]
            
            
        

File: 알고리즘/2/Reverse String.cpp
class Solution {
public:
    void reverseString(vector<char>& s) {
        for(int i=0; i<s.size()/2; i++){
            int pointer = s.size()-1-i;
            swap(s[i], s[pointer]);
            // int temp = s[i];
            // s[i] = s[pointer];
            // s[pointer] = temp;
        }
    }
};